### PR TITLE
Redundant pcc unseal text

### PR DESCRIPTION
**Pull request by tariqkurd-repo**
remove redundant exception check for the PCC being sealed

### CODE REVIEW

**Review by francislaus — CHANGES_REQUESTED** [review]
This does not match the result of the discussion in #112 if I understand your PR correctly.

**Comment by tariqkurd-repo** [comment]
@francislaus this PR is saying that there can't be an exception caused by the PCC being sealed, so I've removed that from the exception table

**Review by andresag01 — APPROVED** [review]

File: src/riscv-integration.adoc [hunk 1]
```diff
@@ -1090,6 +1090,8 @@ NOTE: `auth_cap` is <<ddc>> for Legacy mode and `cs1` for Capability Mode
 | capability stores                              | 6                    | N/A                     | N/A                        |capability alignment  | Misaligned capability store
 |=========================================================================================
 
+^1^ this check is architecturally required, but is not required in an implementation
```
Comment (arichardson) [inline src/riscv-integration.adoc:1093]:
```suggestion
^1^ This check is architecturally required, but is impossible to encounter so may not required in an implementation.
```

Maybe like this?

**Review by arichardson — APPROVED** [review]
LGTM with the footnote changes to a full sentence.

**Pull request merged**